### DISCUSSION TITLE
File is imported but still not defined

### DISCUSSION THREAD

**Discussion by aj3423** [hyperledger-labs/solang#1171] [Q&A] [answered]
A simple contract that imports `ERC20`:
```solidity
// SPDX-License-Identifier: MIT
pragma solidity > 0.8.0;

import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";

contract Test is ERC20 {
}
```
Compile it with solang:
> solang compile --target solana -I /usr/local/lib/node_modules/ test.sol

The `ERC20.sol` file exists and it's imported successfully, but it still shows error:
```
error: 'ERC20' not found
  ┌─ /e/sol/1/test.sol:6:18
  │
6 │ contract Test is ERC20 {
  │                  ^^^^^
```

Is this a bug? I'm using the latest version 0.2.1.

**Comment by seanyoung** [comment]
I can't reproduce this. Can you give a copy of your `ERC20.sol` please

**Comment by aj3423** [comment]
@seanyoung I just tried the new release 0.2.2, same problem. 
The test file: [test.sol.zip](https://github.com/hyperledger/solang/files/10795022/test.sol.zip)
The `ERC20.sol`: [ERC20.sol.zip](https://github.com/hyperledger/solang/files/10795122/ERC20.sol.zip)


My compile command:
```
solang compile --target solana -I /usr/local/lib/node_modules/ test.sol
```
The result:
![image](https://user-images.githubusercontent.com/4710875/220388515-6e2dc5e7-17ab-48a7-99df-60e9e77877ef.png)


The `@openzeppelin` contracts and the `ERC20.sol` do exist, the `solc` works fine:
```
/e/sol/2 > solc --base-path . --include-path /usr/local/lib/node_modules/ test.sol
Compiler run successful. No output generated.
```

**Comment by seanyoung** [comment, marked as answer]
The issue is that ERC20 needs work for Solana, see #1174
